go.mod

```diff
@@ -9,8 +9,8 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/hashicorp/consul-awsauth v0.0.0-20220713200329-83146e702996
 	github.com/hashicorp/consul-server-connection-manager v0.1.2
-	github.com/hashicorp/consul/api v1.10.1-0.20230830173140-255aa158db8c
-	github.com/hashicorp/consul/sdk v0.14.1
+	github.com/hashicorp/consul/api v1.26.1-rc1
```

Comment:
Most of these seem to be pointing to release candidate versions. Should this change be released along with the GA release of `0.7.0`?

Reply:
we synced offline and this will be update with 1.17 release before releasing `0.7.0`